v2g-liberty/rootfs/root/appdaemon/apps/v2g_liberty/fm_client.py

```diff
@@ -67,18 +66,30 @@ def __init__(self, hass: Hass):
             + self.DELAY_FOR_INITIAL_ATTEMPT
         )
 
-        # Ping every half hour. If offline a separate process will run to increase polling frequency.
+        # Ping every half hour. If offline, a separate process will run to increase frequency.
         self.connection_error_counter = 0
         # self.run_every(self.ping_server, "now", 30 * 60)
         self.handle_for_repeater = ""
 
     async def test_fm_connection(self, host_url, username, password):
+        """Test if charger can connect with given host and port
```

Comment:
Something to consider: 
_Uncle Bob_ is a legend in the software development world, and author of the Clean Coding series of books/videos. He promotes writing as little comments as possible, as they are notoriously difficult to maintain. Instead, write your code so cleanly that you do not need the comment.
With this in mind, I wonder if you really need to add comments like this.

Reply:
I agree with Uncle Bob here but the Linter complains... Can that just be ignored or can/should the linter be configured differently?

Reply:
We can certainly ignore it. And if we find it annoying, we should figure out how to silence the linter.

Reply:
But in this case it also documents the somewhat complex returned values , so i think it still is a good thing to have a docstring here.
I do agree there are some docstrings just for the linter...